notebooks/Deploy embed v3 model.ipynb

```diff
@@ -83,8 +83,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# TODO: replace with the cohere embed v3 model packages when ready\n",
-    "cohere_package = \"cohere-english-small-v2-2-5c5864d6b4ee3277bbbca80225b5638f\"\n",
+    "\"\"\"\n",
+    "English: cohere-embed-english-v3-0-894d9aa7482539ea9c1d8c06097a37ed\n",
+    "English Light: cohere-embed-english-light-v3--72a145a6b70135d1a488a3c3a4c1d8a7\n",
+    "Multilingual: cohere-embed-multilingual-v3-0-ac06ff5727013b1b8d1f8551eac4d1bd\n",
+    "Multilingual Light: cohere-embed-multilingual-ligh-4e098657798e3be2aab41121c746a18a\n",
```

Comment:
is there `t` missing?
```suggestion
    "Multilingual Light: cohere-embed-multilingual-light-4e098657798e3be2aab41121c746a18a\n",
```

Reply:
not sure - ccing: @andrewvonrosenbach

Reply:
not a typo - t is not missing. see other comment thread.